ci/libbuild.sh

```diff
@@ -21,7 +21,7 @@ make() {
 
 build() {
     env NOCONFIGURE=1 ./autogen.sh
-    ./configure --prefix=/usr --libdir=/usr/lib64 "$@"
+    ./configure --prefix=/usr --libdir=/usr/lib64 --sysconfdir=/etc "$@"
```

Comment:
Oh man that's evil.  Right.  We should think about trying to have a canonical upstream for `libbuild.sh` too but not a blocker for this.